tests/middleware/test_base.py

```diff
@@ -15,6 +15,8 @@
 from starlette.testclient import TestClient
```

Comment:
I'd prefer if we maintain the tests unchanged, and unpack the `ExceptionGroup` on the `BaseHTTPMiddleware` in case we have a single exception.